app/src/main/java/com/nervousfish/nervousfish/modules/pairing/AbstractPairingHandler.java

```diff
@@ -4,12 +4,12 @@
  * Contains common methods shared by all pairing modules.
  */
 @SuppressWarnings("PMD.AbstractClassWithoutAbstractMethod")
-abstract class APairingHandler {
+abstract class AbstractPairingHandler {
```

Comment:
That doesn't really sound like a good argument to me... What is the issue if you enable it? Does it give you a regexp to which the name should comply? In that case you can just alter the regexp to allow for the naming scheme `APairingHandler`.